src/features/view-subdomains/SubdomainTableRow/SubdomainTableRow.tsx

```diff
@@ -65,25 +60,26 @@ export const SubdomainTableRow: FC<SubdomainTableRowProps> = ({
 						<span>0://{zna}</span>
 					</div>
 				</div>
-			</TableData>
-			<TableData alignment={'right'} className={styles.Metrics}>
+			</Cell>
+			<Cell alignment={'right'} className={styles.Metrics}>
 				<SkeletonText
 					asyncText={{
 						text: volume,
 						isLoading: isLoadingMetrics,
 					}}
 				/>
-			</TableData>
+			</Cell>
 
-			{!isOwnedByUser && (
-				<TableData alignment={'right'} className={styles.Button}>
-					{buyNowPrice ? (
+			<Cell alignment={'right'} className={styles.Button}>
+				{!isOwnedByUser &&
```

Comment:
Assuming this switch is because we still want to render the empty cell?

Reply:
Correct!